tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CiamIntegrationTests.cs

```diff
@@ -144,6 +149,74 @@ public async Task ClientCredentialWithClientSecret_Ciam_Async(string authorityFo
             Assert.AreEqual(TokenSource.Cache, result.AuthenticationResultMetadata.TokenSource);
         }
 
+        [TestMethod]
+        public async Task Ciam_Cud_Obo_Test_Async()
+        {
+            string authorityNonCud = "https://MSIDLABCIAM6.ciamlogin.com";
+            string authorityCud = "https://login.msidlabsciam.com/fe362aec-5d43-45d1-b730-9755e60dc3b9/v2.0/";
```

Comment:
So CUD is custom user domain?

What are all the forms custom and non-custom domains can be?

Reply:
That format is the only one for CIAM CUD (host/tenant/v2.0). But we're not parsing it.

For normal CIAM (non-CUD), the 3 formats remain. `https://tenant.ciamlogin.com`, `https://tenant.ciamlogin.com/tenant`, `https://tenant.ciamlogin.com/guid`